tensorflow/core/kernels/gather_nd_op_cpu_impl.h

```diff
@@ -32,6 +32,8 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/util.h"
 
+#define TF_CALL_DATASET_TYPES(m) TF_CALL_ALL_TYPES(m) TF_CALL_QUANTIZED_TYPES(m)
```

Comment:
if this is a single registration for the one line below; i'd rather you just call ALL_TYPES and QUANTIZED_TYPES at L156